dashboard/lib/tasks/seed.rake

```diff
@@ -48,7 +48,7 @@ namespace :seed do
   end
 
   timed_task foorm_libraries: :environment do
-    Foorm::LibraryQuestion.setup
+    Foorm::Library.setup
```

Comment:
semantically this makes more sense, which i think is another sign this was a good move!